exercises/farewell-workflow/solution/Client/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Run Workflow file
+using Temporalio.Client;
+using Temporalio.Farewell.Workflow;
+
+// Connect to the Temporal server
+var client = await TemporalClient.ConnectAsync(new("localhost:7233") { Namespace = "default" });

```

Comment:
Inconsistent with other exercises like #5 on whether you explicitly specify the `"default"` namespace or not